src/tools/tidy/src/style.rs

```diff
@@ -444,7 +444,7 @@ pub fn check(path: &Path, bad: &mut bool) {
                 suppressible_tidy_err!(err, skip_cr, "CR character");
             }
             if filename != "style.rs" {
-                if trimmed.contains("TODO") {
+                if trimmed.contains("TODO") && !trimmed.contains("ignore-tidy-todo") {
```

Comment:
Add a comment here documenting the use case of `ignore-tidy-todo` to allow suggesting a TODO comment in diagnostics